tests/lib/rules/no-magic-numbers.js

```diff
@@ -719,6 +729,14 @@ ruleTester.run("no-magic-numbers", rule, {
             errors: [
                 { messageId: "noMagic", data: { raw: "100" }, line: 1 }
             ]
+        },
+        {
+            code: "const { param = 123 } = sourceObject;",
+            options: [{ ignoreDefaultValues: false }],
+            env: { es6: true },
+            errors: [
+                { messageId: "noMagic", data: { raw: "123" }, line: 1 }
+            ]
```

Comment:
We should also test the option's default value, e.g. one test without `options`, and another test with `options: [{}]`

Reply:
Fixed.